packages/react-components/tests/browser/ImageOverlay.spec.tsx

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+import { expect } from '@playwright/experimental-ct-react';
+import { test as betaTest } from './FlavoredBaseTest';
+import { ImageOverlay, Message, MessageThread } from '../../src';
+
+betaTest.describe('ImageOverlay tests', () => {
+  betaTest.skip(({ isBetaBuild }) => !isBetaBuild, 'The tests should be run for beta flavor only');
+  const messages: Message[] = [
+    {
+      messageType: 'chat',
+      senderId: 'user3',
+      content:
+        '<p>How should I design my new house?</p><p><img alt="image" src="images/inlineImageExample1.png" itemscope="png" width="166.5625" height="250" id="SomeImageId1" style="vertical-align:bottom"></p><p><img alt="image" src="images/inlineImageExample2.png" itemscope="png" width="374.53183520599254" height="250" id="SomeImageId2" style="vertical-align:bottom"></p><p>&nbsp;</p>',
+      senderDisplayName: 'Miguel Garcia',
+      messageId: Math.random().toString(),
+      createdOn: new Date('2019-04-13T00:00:00.000+08:09'),
+      mine: false,
+      attached: false,
+      contentType: 'html'
+    },
+    {
+      messageType: 'chat',
+      senderId: 'user2',
+      senderDisplayName: 'Robert Tolbert',
+      messageId: Math.random().toString(),
+      content: 'Cool, I love the second one!',
+      createdOn: new Date('2019-04-13T00:00:00.000+08:10'),
+      mine: true,
+      attached: false,
+      contentType: 'text'
+    }
+  ];
+
+  betaTest('ImageOverlay should be shown correctly when loading', async ({ mount }) => {
+    const component = await mount(
+      <div>
+        <MessageThread userId={'1'} messages={messages} />
```

Comment:
Do we need to mount MessageThread here?

Reply:
> Do we need to mount MessageThread here?

ImageOverlay might be a special component since it's a Modal?
I've tested without the MessageThread component and the test fails with the error below:

```
      - expect.toHaveScreenshot(image-overlay-loading-stage.png) with timeout 5000ms
      -   generating new stable screenshot expectation
      - waiting for locator('#root').locator('internal:control=component')
      -   locator resolved to <div class="css-109 body-188">…</div>
      - taking element screenshot
      -   disabled all CSS animations
      -   waiting for element to be visible and stable
      -     element is not visible - waiting...
      - Timeout 5000ms exceeded.
```